### PR TITLE
 SONARPY-972 Remove Typeshed parsing logic 

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**

### CODE REVIEW

File: python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java [hunk 1]
```diff
@@ -273,11 +269,9 @@ public void visitFileInput(FileInput tree) {
       createScope(tree, null);
       enterScope(tree);
       moduleScope = currentScope();
-      if (!SymbolUtils.isTypeShedFile(pythonFile) || !BASE_MODULES.contains(pythonFile.fileName())) {
-        Map<String, Symbol> typeShedSymbols = TypeShed.builtinSymbols();
-        for (String name : BuiltinSymbols.all()) {
-          currentScope().createBuiltinSymbol(name, typeShedSymbols);
-        }
+      Map<String, Symbol> typeShedSymbols = TypeShed.builtinSymbols();
+      for (String name : BuiltinSymbols.all()) {
```
Comment (andrea-guarino-sonarsource) [inline python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java:273]: probably we don't need anymore to go through `BuiltinSymbols.all()` , however let's do that in a separate PR :)

**Review by andrea-guarino-sonarsource — APPROVED** [review]
LGTM! 
I think there two additional things to consider:
- I believe we could (should?) keep the `LICENSE` file from typeshed
- Can we reduce the size constraints of plugin jar?

**Comment by guillaume-dequenne-sonarsource** [comment]
> LGTM! I think there two additional things to consider:
> 
>     * I believe we could (should?) keep the `LICENSE` file from typeshed
> 
>     * Can we reduce the size constraints of plugin jar?

With regards to the license file, we still have it through our submodule of Typeshed in the serializer, so I think that should be enough?
Also unfortunately, plugin size still sits nicely within the constraints at 8.7MB (8.3 - 9.3 being the current range).
